packages/invoices/src/OutboundShipment/DetailView/modals/BatchesTable.tsx

```diff
@@ -1,64 +1,32 @@
 import React from 'react';
 import {
+  NumericTextInput,
```

Comment:
I shouldn't have done this - I had just noticed the `register` wasn't really doing much so I swapped to use `NumericTextInput` and removed the use of the form context thing

Reply:
it was used mainly for state management and error messaging, so I think not required now. neater to use the `NumericTextInput`